docs/self-audit/scripts/self-audit.sh

```
@@ -0,0 +1,460 @@
+# ====================================================================
+# 統合レポート生成
+# ====================================================================
+
+log_info "統合レポート生成中..."
+
+node scripts/audit-report-generator.js \
+    --input-dir "$OUTPUT_DIR/results" \
+    --output-dir "$OUTPUT_DIR/reports" \
+    --format "$OUTPUT_FORMAT" \
+    --timestamp "$TIMESTAMP" \
+    $([ "$VERBOSE" = true ] && echo "--verbose") \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)